### PR TITLE
Update _dropdown.scss

### PR DESCRIPTION
**Pull request by Ishwari8104**
Action buttons are easy to miss in list views for high contrast mode issue

### CODE REVIEW

File: client/scss/components/_dropdown.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .w-dropdown__toggle {
   @apply w-px-2 w-text-text-label w-bg-transparent;
+  @media (prefers-contrast: more) {
+    @apply w-border w-border-solid w-border-text-label; 
+  }
+}
```
Comment (laymonage) [inline client/scss/components/_dropdown.scss:6]: This is currently throwing a syntax error due to the extraneous closing curly brace.

```suggestion
```

Also, could you add some screenshots to verify the issue you are fixing?

**Review by laymonage — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by lb-** [comment]
Screenshot 2024-09-29 at 4 10 58 pm

I can confirm this issue exists, see the '...' button toggle.

**Comment by lb-** [comment]
@Ishwari8104  thanks for flagging this, I opted for a simpler fix (just adding a 2px transparent border), this will ensure the border gets colours for high contrast mode.

**Screenshot after fix**


Screenshot 2024-09-29 at 4 19 34 pm


I have added you to the contributors list, I could not find a name so have used `Ishwari8104`, let me know if you want a different name listed.

A few suggestions for future PRs

1. Have a full read through our first contribution guide, it covers a lot of common learning items that can help your PRs https://docs.wagtail.org/en/stable/contributing/first_contribution_guide.html
2. We strongly recommend including a screenshot of the problem you encountered (especially if it's a UI/ High contrast item), this makes reviewing way faster for us.
3. Always be sure to check the linting / formatting locally before doing a PR, it will help the code run through the CI fully.

However, we understand it's a big codebase to get your head around so well done on finding the place to fix this and submit your first contribution to Wagtail.

Thanks for making things better for high contrast users.

**Review by lb- — APPROVED** [review]

**Pull request merged**